### PR TITLE
Add vssyncd debian package

### PR DESCRIPTION
**Pull request by kcudnik**

### CODE REVIEW

**Comment by lguohan** [comment]
do you also need to add the package in the control file?

File: debian/vssyncd.debhelper.log [hunk 1]
```diff
@@ -0,0 +1,3 @@
+dh_autotools-dev_updateconfig
+override_dh_auto_configure dh_auto_configure
+dh_auto_configure
```
Comment (lguohan) [inline debian/vssyncd.debhelper.log]: this log file is not needed.

**Comment by qiluo-msft** [comment]
Retest this please.

**Comment by qiluo-msft** [comment]
Retest this please.

**Comment by qiluo-msft** [comment]
Retest this please.

**Comment by qiluo-msft** [comment]
Retest this please.

File: debian/control [hunk 2]
```diff
@@ -18,6 +18,12 @@ Description: This package contains sync daemon with rpc for SONiC project.
   This sync daemon syncs the ASIC_DB in Redis database and the real ASIC via SAI.
   This daemon contains saithrift rpc library for remote control of an ASIC.
 
+Package: vssyncd
+Architecture: any
+Conflicts: syncd-rpc
```
Comment (lguohan) [inline debian/control]: should also add syncd as conflict

File: debian/vssyncd.install [hunk 3]
```diff
@@ -0,0 +1 @@
+usr/bin/vssyncd
```
Comment (lguohan) [inline debian/vssyncd.install]: usr/bin/vssyncd usr/bin/syncd

**Pull request merged**